src/webgpu/web_platform/reftests/canvas_colorspace.html.ts

```diff
@@ -0,0 +1,68 @@
+import { runRefTest } from './gpu_ref_test.js';
+
+function bgraFromRgba(rgba: Uint8Array) {
+  const bgra = rgba.slice();
+  for (let i = 0; i < bgra.length; i += 4) {
```

Comment:
`i` is not used. Fixing this makes the results look correct on my system (at least, the ones that work)

optional: Could be written `[bgra[i], bgra[i + 2]] = [bgra[i + 2], bgra[i]];` if you like that style

Reply:
doh! thanks for catching that!